github/src/main/kotlin/gropius/sync/github/SyncNotificator.kt

```diff
@@ -0,0 +1,63 @@
+package gropius.sync.github
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import gropius.model.architecture.IMS
+import gropius.model.architecture.IMSIssue
+import gropius.model.architecture.IMSProject
+import gropius.model.common.ExtensibleNode
+import gropius.model.template.MutableTemplatedNode
+import kotlinx.coroutines.reactor.awaitSingle
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.data.neo4j.core.ReactiveNeo4jOperations
+import org.springframework.data.neo4j.core.findById
+import org.springframework.stereotype.Component
+
+/**
+ * Dummy class for the future notification center
+ * @param helper Reference for the spring instance of JsonHelper
+ * @param neoOperations Reference for the spring instance of ReactiveNeo4jOperations
+ */
+@Component
+class SyncNotificator(
```

Comment:
I know this is a dummy class, but at least the functions which will be kept (although with other implementation) should get some docs

Reply:
97e7d2bad00bc5df055901a6122ebc81ba380105